front/src/weekview/Weekview.module.scss

```diff
@@ -31,64 +33,64 @@
   border-top: 1px solid;
 }
 
-.flex-container2 ~ .flex-container2 {
+.flexContainer2 ~ .flexContainer2 {
   border-top: 0px;
 }
 
-.date-header {
+.dateHeader {
   height: 10em;
   display: flex;
   align-items: center;
   justify-content: center;
   text-align: center;
 }
 
-.flex-container2 > a {
-  outline: 0px solid black;
-  background-color: #eae7dc;
+.flexContainer2 > a {
+  outline: 0px $COLOR_BLACK;
+  background-color: $COLOR_CREAM10;
   width: calc(100% / 7);
   margin: 0px;
   text-align: center;
   line-height: 30px;
   font-size: 20px;
   text-decoration: none;
-  color: black;
+  color: $COLOR_BLACK;
 }
 
-.flex-container > a {
+.flexContainer > a {
   border-right: 1px solid;
   width: calc(100% / 7);
   margin: 0px;
   text-align: center;
   height: 125px;
   text-decoration: none;
-  color: black;
+  color: $COLOR_BLACK;
 }
 
-.main-grid {
+.mainGrid {
   direction: column;
   justify-content: left;
   align-items: center;
 }
 
-.arrow-right {
+.arrowRight {
```

Comment:
I think the arrowRight/Left styles get applied from shared already, so I'd only leave styles that need to be overridden here (which are probably only the margin-right/left)